go/kbfs/libkbfs/subscription_manager.go

```diff
@@ -37,6 +38,14 @@ func getCleanInTlfPath(p *parsedPath) cleanInTlfPath {
 	return cleanInTlfPath(path.Clean(p.rawInTlfPath))
 }
 
+func getParentPath(p cleanInTlfPath) (parent cleanInTlfPath, ok bool) {
```

Comment:
Wouldn't `path.Split()` do this same thing, and you can just check for a `""` parent instead of using a bool?

Reply:
Thanks Strib! I was thinking finding last `/` would be slightly less wasteful than split-then-join. An explicit bool is just to make it ... explicit. I'm gonna merge this for now since once we have actual topic checks these will all likely change.

Reply:
@songgao 	`path.Split` is different from `strings.Split` -- it does exactly what you're done here, except it doesn't return a bool.  https://golang.org/src/path/path.go?s=3743:3785#L137

Reply:
Ooh! I missed that. Thanks

Reply:
And clever use of the `-1` returned value!

Reply:
Actually `path.Split` is different. It includes the `/` at the end of parent dir. I guess we could do a `path.Clean`.